### ISSUE TITLE
Maria's song plays too loudly / chvol being called incorrectly

### ISSUE THREAD

**Issue by drojf** [07th-mod/umineko-question#15]
So I was looking in the script where `chvol` was used, and it turns out it's always called AFTER a `dwave` command (`dwave` or `dwaveloop`), not before. This means that:

```
;Maria's song start
chvol 49,5 ;Maria's song volume
dwaveloop 49, mar_1e859_1
```

should actually be 

```
;Maria's song start
dwaveloop 49, mar_1e859_1
chvol 49,5 ;Maria's song volume
```

I tested this and the second one works correctly. This also will fix the cases where multiple voices are played at once and the voices are unequal (if that's still an issue anymore)

**Comment by ItaloKnox** [comment]
Fixed this one in particular, any ideas on how to narrow down similar issues?

**Comment by drojf** [comment]
I don't think there should be any other issues? the only time `chvol` affects anything is if you play a sound using `dwave` or `dwaveloop`. 

`dwaveloop` is only used twice in the script, once for maria's song, and once in one of the functions which plays music effects. But the second usage would have been part of the original game, so shouldn't have any problems.

```
Line 85071: dwaveloop 49, mar_1e859_1
Line 310954:   dwaveloop %Me_Play_Channel,$Me_Play
```

As for the `dwave` function, a list of all the cases where dwave is called WITHOUT using channel zero is here:

https://gist.github.com/drojf/a28f8bf8834f0088ac23b9691bfaa190

All instances of `chvol` are also listed - they are only used in a custom function (as part of `meplay`) which play effects like the howling wind noise, and they seem to work correctly.

So assuming my searching criteria / method has been done correctly, there shouldn't be any other cases to check.

**Comment by DoctorDiablo** [comment]
I think you covered all the scenarios.  The only special cases I encountered while adding the voice lines were those 10 dwave commands that use multiple channels, and Maria's song.

**Comment by ChrisE87** [comment]
I also noticed parts where the rain is suddenly a lot louder in comparison to the voices. Not really sure where that's coming from, but my guess is that one of the channels is louder than the rest and depending on the build-up of the scene, the parts use different channels. Is this also covered by the suggested solutions?

**Comment by drojf** [comment]
Yea, that has already been reported here https://github.com/07th-mod/umineko-question/issues/1#issuecomment-249711844. In the script, change default volume to 70 and voice vol to 60 and it shouldn't be as bad. It's due to the fact that in the script, sometimes it sets the volume of things to say, 70, or 100, without caring about the 'default vol' variable.

@ItaloKnox - in the latest release, it appears you didn't set the volume higher. I tried it out at default volume 70 (the default) and voice volume at 60 and it is more or less OK, I think we should set it to that for now, until we come up with a better fix.

**Comment by drojf** [comment]
Fixed this on v1.1 (just maria's song part, not the other issues)

**Comment by ItaloKnox** [comment]
@drojf You're right, somehow I forgot about this. I'll change it and update the script without doing a new release.
